src/components/search/omnibox-directions.component.tsx

```diff
@@ -247,14 +257,32 @@ const OmniboxDirections = ({
             <TouchableOpacity>
               <FontAwesome name="car" size={24} style={styles.travelModeIcon} />
             </TouchableOpacity>
-            <TouchableOpacity>
+            <TouchableOpacity
+              style={
+                travelMode === TravelMode.ACCESSIBLE
+                  ? styles.travelModeButtonSelected
+                  : null
+              }
+              onPressOut={() => {
+                setTravelMode(TravelMode.ACCESSIBLE);
+              }}
+            >
               <FontAwesome
                 name="wheelchair"
-                size={24}
+                size={26}
```

Comment:
Could be moved to a constant

Reply:
I will not move this since this is not part of my ticket. I only change this size to make the wheelchair Icon look bigger.